src/main/java/com/ratifire/devrate/validation/annotations/ValidFeedbackType.java

```diff
@@ -0,0 +1,40 @@
+package com.ratifire.devrate.validation.annotations;
+
+import com.ratifire.devrate.validation.validators.FeedbackTypeValidator;
+import jakarta.validation.Constraint;
+import jakarta.validation.Payload;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Custom annotation for validating feedback types.
+ */
+@Constraint(validatedBy = FeedbackTypeValidator.class)
+@Target({ElementType.FIELD, ElementType.PARAMETER})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface ValidFeedbackType {
+
+  /**
+   * The default validation message.
+   *
+   * @return the default message
+   */
+  String message() default "Invalid feedback type. "
+      + "It can be only 'PROPOSITION', 'ISSUE', or 'FEEDBACK'.";
```

Comment:
and what if we add a new feedback type? We need to always remember to update this file when we update the enum. Need to be reworked